### DISCUSSION TITLE
Site replication

### DISCUSSION THREAD

**Discussion by LuisVilaplana** [oqtane/oqtane.framework#3309] [General]
Ok, let's suppose I already have a fully configured site, with its pages, modules, themes, etc. How do I replicate it on other sites? I love Oqtane's Multitenant capability, but I don't think it's productive to manually recreate everything from scratch for each new site I create. 

I'm thinking of something like a SaaS where each client enters their own application with their own database, completely separated from other clients, but it's the same application as other clients.

**Comment by sbwalker** [comment]
There 3 different ways to accomplish this...

https://www.oqtane.org/blog/!/62/routable-modules
